hwy/ops/x86_512-inl.h

```diff
@@ -1164,15 +1164,33 @@ HWY_API Vec512<uint16_t> operator*(Vec512<uint16_t> a, Vec512<uint16_t> b) {
 HWY_API Vec512<uint32_t> operator*(Vec512<uint32_t> a, Vec512<uint32_t> b) {
   return Vec512<uint32_t>{_mm512_mullo_epi32(a.raw, b.raw)};
 }
-
+HWY_API Vec512<uint64_t> operator*(Vec512<uint64_t> a, Vec512<uint64_t> b) {
+  return Vec512<uint64_t>{_mm512_mullo_epi64(a.raw, b.raw)};
+}
+HWY_API Vec256<uint64_t> operator*(Vec256<uint64_t> a, Vec256<uint64_t> b) {
```

Comment:
FYI normally we would define Vec256 and Vec128 overloads in x86_256/128, but this seems fine as-is because these overloads are only defined when we're targeting AVX3 or AVX3_DL, i.e. this header is included.

Reply:
Yes, they may be 256 or 128 bits, but they're AVX512 instructions / they require AVX3 and thus I didn't want them defined without it.

Reply:
Right. Usually the way we specialize AVX3 implementations of 256-bit vectors is to have one function in x86_256 with `#if HWY_TARGET <= HWY_AVX3` inside, but as mentioned it's fine as-is in this case.